composer.json

```diff
@@ -18,8 +18,8 @@
     }
   },
   "require": {
-    "php":              "^7.2",
-    "league/flysystem": "^1.0"
+    "php":              "^7.2||^8.0",
+    "league/flysystem": "^1.0||^2.0"
```

Comment:
This library cannot be compatible with `league/flysystem:^2.0` Since this is a plugin, which have been removed in flysystem 2.

Reply:
i see.
reverted the change.